_tools/syncfiles.txt

```diff
@@ -1,6 +1,8 @@
 api/.htaccess
 api/versioninfo.php
 api/historydata.php
+api/keyboard.php
+api/util.php
```

Comment:
This impacts the [Files to Upload](https://github.com/keymanapp/downloads.keyman.com#files-to-upload) section of README.md